examples/js/cli.js

```diff
@@ -213,6 +257,7 @@ function printUsage () {
     log ('node', process.argv[1], 'okcoin fetchOHLCV BTC/USD 15m')
     log ('node', process.argv[1], 'bitfinex fetchBalance')
     log ('node', process.argv[1], 'kraken fetchOrderBook ETH/BTC')
+    log ('node', process.argv[1], 'binanceusdm fetchTrades BTC/USDC undefined --param until=1746988377067')
```

Comment:
this example is not correct right? we're missing the `limit` placeholder, undefined in this case, otherwise  limit = {"until":"1746988377067"}

Reply:
@olegtaranenko actually I removed this requirement here: https://github.com/ccxt/ccxt/pull/25975